dali/kernels/signal/dct/dct_args.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_SIGNAL_DCT_DCT_ARGS_H_
+#define DALI_KERNELS_SIGNAL_DCT_DCT_ARGS_H_
+
+namespace dali {
+namespace kernels {
+namespace signal {
+namespace dct {
+
+/**
+ * @brief DCT kernel arguments
+ */
+struct DctArgs {
+  /// @brief DCT type. Supported types are 1, 2, 3, 4
```

Comment:
What this value maps to?

Reply:
It doesn't map to anything. See:
https://en.wikipedia.org/wiki/Discrete_cosine_transform#Formal_definition

Reply:
So this reference should be put here.

Reply:
It's in the operator documentation, but ok, I'll add it here as well